pom.xml

```diff
@@ -77,7 +77,7 @@ THE SOFTWARE.
 
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.452.4</jenkins.version>
+        <jenkins.version>2.462.2</jenkins.version>
```

Comment:
I guess this will require a bump in the BOM too https://github.com/jenkinsci/ec2-plugin/blob/master/pom.xml#L204-L205

Reply:
Done: 2f073ac840be7379884f376b5b87dabba395d596